webdriver/tests/bidi/script/call_function/call_function.py

```diff
@@ -0,0 +1,155 @@
+import pytest
+
+from webdriver.bidi.modules.script import ContextTarget, ScriptEvaluateResultException
+from ... import recursive_compare, any_string, any_int
+from .. import any_stack_trace
+
+
+@pytest.mark.asyncio
+async def test_invalid_function(bidi_session, top_context):
+    with pytest.raises(ScriptEvaluateResultException) as exception:
```

Comment:
I think that steps 13 and 14 of the spec say that if the function_declaration is invalid, then it should throw an invalid argument error. Can you check?

> Let function body evaluation status be the result of [evaluate function body](https://w3c.github.io/webdriver-bidi/#evaluate-function-body) (function declaration, environment settings, base URL, and options).
> 
> If function body evaluation status.[[Type]] is throw, return [error](https://w3c.github.io/webdriver/#dfn-errors) with [error code](https://w3c.github.io/webdriver/#dfn-error-code) [invalid argument](https://w3c.github.io/webdriver/#dfn-invalid-argument).

Reply:
Thanks for pointing this out! I fixed the spec: https://github.com/w3c/webdriver-bidi/pull/242